action.yml

```diff
@@ -68,18 +75,45 @@ runs:
         sudo clamscan --infected github-release-assets/ > scan-results.log
         echo "CLAMAV_RETURN_CODE=$?" >> $GITHUB_ENV
         exit 0;
-    # - name: Notify Slack on viruses detected
-    #   if: ${{ env.CLAMAV_RETURN_CODE == '1' }}
-    #   uses: 8398a7/action-slack@v3.9.3
-    #   with:
-    #     username: ClamAV Virus Scanning Workflow
-    #     status: failure
-    #     text: "ClamAV has detected a virus in the release at ${{ env.RELEASE_HTML_URL }}"
-    #     author_name: ""
-    #     fields: repo,ref,action,commit,author
-    #     icon_emoji: ":biohazard_sign:"
-    #   env:
-    #     SLACK_WEBHOOK_URL: ${{ secrets.RELEASE_ANTIVIRUS_SLACK_WEBHOOK_URL }}
+    - name: Notify Slack on viruses detected
+      shell: pwsh
+      env:
+        SLACK_TOKEN: ${{ inputs.slack-token }}
+        SLACK_CHANNEL: ${{ inputs.slack-channel }}
+      run: |
+        # Notify Slack on viruses detected
+        
+        if ($Env:SLACK_TOKEN -eq '') {
+          Write-Output "inputs.slack-token not provided, can't post to Slack"
+          exit 0
+        }
+        if ($Env:SLACK_TOKEN -eq '') {
+          Write-Output "inputs.slack-channel not provided, can't post to Slack"
+          exit 0
+        }
+        if ($Env:CLAMAV_RETURN_CODE -eq '0') {
```

Comment:
Seems like maybe this should be the thing that is checked for first. I'm thinking about the scenario where you don't have slack configured and there are no viruses. 

The way things are now, you'll get one of the `inputs.slack-token not provided, can't post to Slack`, log entries instead of the the one that actually matters for that scenario, that there is no reason to alert in the first place.